### PR TITLE
Prepare branch renaming to `main`

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

`master` naming for repos is deprecated in the tech industry.
`main` is now the popular choice, and it would be easier for devs that openfun default branches be named `main`.

## Proposal

- 🧑‍💻(project) prepare branch renaming from master to main
- 📝(project) rename `main` doc version to `dev`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -488,9 +488,9 @@ jobs:
             # - DOCS_VERSION: 1.1 (for git tag v1.1.2)
             # - DOCS_ALIAS: latest
             # or
-            # - DOCS_VERSION: master (for master branch)
+            # - DOCS_VERSION: dev (for main branch)
             # - No DOCS_ALIAS
-            DOCS_VERSION=$([[ -z "$CIRCLE_TAG" ]] && echo $CIRCLE_BRANCH || echo ${CIRCLE_TAG} | sed 's/^v\([0-9]\.[0-9]*\)\..*/\1/')
+            DOCS_VERSION=$([[ -z "$CIRCLE_TAG" ]] && echo "dev" || echo ${CIRCLE_TAG} | sed 's/^v\([0-9]\.[0-9]*\)\..*/\1/')
```
Comment (quitterie-lcs) [inline .circleci/config.yml:493]: Why do you change to echo "dev"?

Comment (wilbrdt) [reply .circleci/config.yml:493]: I think a "main" version for the docs would be confusing (whereas no confusion with "master").
I personally find a "dev" version for the docs to be straight to the point. WDYT?

Comment (quitterie-lcs) [reply .circleci/config.yml:493]: You're right! That's better !

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**